packages/core/src/components/ui/ProductGallery/ProductGalleryPage.tsx

```diff
@@ -34,4 +50,4 @@ function GalleryPage({ page, title, productCard }: Props) {
   )
 }
 
-export default GalleryPage
+export default memo(ProductGalleryPage)
```

Comment:
Why was this introduced?

Reply:
This memo aims to prevent cases when the parent (`ProductGallery`) component is rendered and, in turn, the `ProductGalleryPage` is also rendered, but the props remain the same.

before: 8 rerenders when clicking on the `load more` button per Page.
after: 4 rerenders when clicking on the `load more` button per Page.
(even after memo the `RenderSections`)